pytest_docker_postgres/__init__.py

```diff
@@ -65,3 +79,18 @@ def db_engine(in_docker_compose, docker_services):
     create_database(url)
     yield create_engine(url)
     drop_database(url)
+
+
+@pytest.fixture(scope="function")
+def schema_db_engine(db_engine, load_database):
+    connection = db_engine.connect()
+    path = f"{Path.home()}/{load_database}/"
```

Comment:
why use `Path.home`?

Reply:
for testing :P but we don't need this actually